File: input/java/Circle01.java
import java.lang.Math;
import java.lang.IllegalArgumentException;

public class Circle01 {
    private final int radius;

    public Circle01(int radius) {
        if (radius <= 0) throw new IllegalArgumentException("Radius must be bigger than 0.");
        this.radius = radius;
    }

    public int getRadius() {
        return this.radius;
    }

    public int getDiameter() {
        return this.radius * 2;
    }

    public double getCircumference() {
        return this.radius * Math.PI * 2;
    }

    public double getArea() {
        return Math.pow(this.radius, 2) * Math.PI;
    }
}

File: input/kotlin/Circle01.kt
import java.lang.IllegalArgumentException

class Circle01(radius: Int) {
    val radius: Int
    val diameter: Int
        get() = radius * 2
    val circumference: Double
        get() = radius * Math.PI * 2
    val area: Double
        get() = Math.pow(radius.toDouble(), 2.0) * Math.PI

    init {
        if (radius <= 0) throw IllegalArgumentException("Radius must be bigger than 0.")
        this.radius = radius
    }
}